waves-integration/src/test/scala/com/wavesplatform/dex/WavesIntegrationSuiteBase.scala

```diff
@@ -19,7 +20,7 @@ import org.scalatest.matchers.{MatchResult, Matcher}
 import scala.collection.immutable.Vector
 
 // TODO DEX-994
-trait WavesIntegrationSuiteBase extends AnyFreeSpecLike with Matchers with AllureScalatestContext {
+trait WavesIntegrationSuiteBase extends AnyFreeSpecLike with Matchers with AllureScalatestContext with ScalaFutures {
```

Comment:
Let's override the default patience configuration with the timeout of 1 minute. I think it is enough.